website/conferences/views.py

```diff
@@ -80,17 +80,21 @@ def add_poster_by_email(conference, message):
         )
         if user_created:
             created.append(user)
+            user.is_conference_user = True
```

Comment:
We use the term "meeting" not "conference".

Now, that I think of it, this seems like a good use case for the `system_tags` field (which both Node and User have). Instead of adding an additional field, add the `osf4m` system_tag to the node and user.